backend/app/data_products/service.py

```diff
@@ -246,7 +244,8 @@ def update_data_product_about(
         current_data_product.about = data_product.about
         db.commit()
 
-    def ensure_owner(self, authenticated_user: User, data_product: DataProduct):
+    @staticmethod
```

Comment:
I would not necessarily make this a static method? Even though it does not use `self` ?

Reply:
I just updated it here. It has no impact. Just no extra param is used in the function definition

Reply:
We might need to think about this in a broader picture. e.g. in the `DatasetService` class, there is no method that can't be a `staticmethod` currently. But I don't think that warrants making all of the methods `staticmethod`. Not sure what you think. It does not harm to do it, but there is also limited advantages